python/HeaderService/camera_coords.py

```diff
@@ -162,14 +182,25 @@ def __init__(self, vendor,
 
         self.dimh = SCAN_GEOM[vendor]['dimh']
         self.dimv = SCAN_GEOM[vendor]['dimv']
+        self.ccdax = SCAN_GEOM[vendor]['ccdax']
+        self.ccday = SCAN_GEOM[vendor]['ccday']
 
         # Make this info available as part of the class
         self.CHANNEL = CHANNEL[vendor]
         self.SEGNAME = SEGNAME[vendor]
 
+        # values needed for WCS
+        self.CDELT1 = xpixelscale
+        self.CDELT2 = ypixelscale
+        # if undefined, we assume reference position is in the center
+        # of the sensor.
+        if not crpix1:
```

Comment:
Since crpix1 can have a valid value of 0.0, which is False, I think this code should say
```python
if crpix1 is None:
    ...
```
Same below as well.

Reply:
Thanks I've fixed that.